ament_cpplint/ament_cpplint/main.py

```diff
@@ -144,7 +148,8 @@ def main(argv=sys.argv[1:]):
         else:
             print("Not using '--root'")
         print('')
-        arguments += files
+        arguments += [file for file in files
+                      if any(excl not in file or not excl for excl in args.exclude)]
```

Comment:
Nitpick: according to the style guide to not align wrapped line but instead wrap directly after the open parenthesis / bracket and indent one level.

Reply:
you mean:

```python
      arguments += [file for file in files  if any(
          excl not in file or not excl for excl in args.exclude)]
```

Reply:
```
arguments += [
    f for f in files
    if any(excl not in f or not excl for excl in args.exclude)]
```